src/sbvr-api/permissions.ts

```diff
@@ -1611,7 +1613,18 @@ const getReqPermissions = async (
 	odataBinds: ODataBinds = [],
 ) => {
 	const [guestPermissions] = await Promise.all([
-		getGuestPermissions(),
+		(async () => {
+			if (
+				guestPermissionsInitialized === false &&
```

Comment:
For me it worked even without this.
```suggestion
```
Any reason we prefer having it for root users at all??
W/o it this code path will always be consistent on every run.

Reply:
I did comment on it in https://github.com/balena-io/pinejs/pull/559#discussion_r926744545#559 
>I think it makes sense to limit the special case as much as possible to avoid on-going differences between eg
>```
>	passthrough: {
>		req: rootRead,
>	},
>```
>and
>```
>	passthrough: {
>		req: {
>			user: {
>				id: 0,
>				actor: 0,
>				permissions: ['resource.read'],
>			},
>		},
>	},
>```
>Where the former would not include guest permissions but the latter would despite otherwise being identical